src/test/java/features/89 API Testing.feature

```diff
@@ -66,10 +66,55 @@ Feature: 89 API Testing
     Then I verify the response code equals 404
 
   @89F
-  Scenario: 89E DELETE Header Swagger Test
+  Scenario: 89F DELETE Header Swagger Test
     Given I use the API named Pet Store API
     When I add an api_key header with the value 123
     When I DELETE record 10 from the pet endpoint
     Then I verify the response code equals 200
     When I GET record 10 from the pet endpoint
-    Then I verify the response code equals 404
\ No newline at end of file
+    Then I verify the response code equals 404
+
+  @89G
+  Scenario: 89G Body With Parameters Test
+    Given I use the API named Pet Store API
+    When I initialize the data

```

Comment:
When I initialize the configuration values as follows
"""
id: 10
category_name: puppies
"""